app/views/pages/getting_started2.html.erb

```diff
@@ -8,71 +8,69 @@
   </div>
 </div>
 <div class="container">
-  <div class="row minor-top-spacing">
-    <div class="col-md-12">
-      <p><%= t('.interface1').html_safe %></p>
-      <p class="more-info"><%= t '.interface2' %></p>
-      <p><%= t '.interface3' %></p>
-      <p><%= t '.header' %></p>
-      <p class="text-center"><%= image_tag 'pages/getting_started/interface/dropdown.png' %></p>
-      <ol>
-        <li><%= t '.header2' %></li>
-        <li><%= t '.header3' %></li>
-        <li><%= t('.header4', part6_link: link_to(t('pages.tutorials.part6'), conversations_path)).html_safe %></li>
-        <li><%= t('.header5', part6_link: link_to(t('pages.tutorials.part6'), conversations_path)).html_safe %></li>
-        <li><%= t('.header6', part4_link: link_to(t('pages.tutorials.part4'), connecting_path)).html_safe %></li>
-        <li><%= t '.header7' %>
-          <ul>
-            <li><%= t('.header8', part1_link: link_to(t('pages.tutorials.part1'), sign_up_path)).html_safe %></li>
-            <li><%= t('.header9', part4_link: link_to(t('pages.tutorials.part4'), connecting_path)).html_safe %></li>
-            <li><%= t('.header10', part7_link: link_to(t('pages.tutorials.part7'), finishing_path)).html_safe %></li>
-            <li><%= t '.header11' %></li>
-            <li><%= t '.header12' %></li>
-          </ul>
-        </li>
-      </ol>
-      <p><%= t '.columns' %></p>
-      <p class="text-center"><%= image_tag 'pages/getting_started/interface/stream.jpg' %></p>
-      <ul>
-        <li><%= t('.column1').html_safe %></li>
-        <li><%=(t '.column2', part4_link: link_to(t('pages.tutorials.part4'), connecting_path)).html_safe %></li>
-        <li><%= t '.column3' %></li>
-      </ul>
-      <p><%= t '.views' %></p>
-      <h3><%= t '.stream' %></h3>
-      <p><%= t '.stream_ex' %></p>
-      <ol>
-        <li><%= t '.stream1' %></li>
-        <li><%= t '.stream2' %></li>
-        <li><%= t '.stream3' %></li>
-      </ol>
-      <p><%= t '.stream_ex2' %></p>
-
-      <h3><%= t '.activity' %></h3>
-      <p><%= t '.activity_ex' %></p>
-
-      <h3><%= t '.mentions' %></h3>
-      <p><%= t('.mentions_ex', part4_link: link_to(t('pages.tutorials.part4'), connecting_path)).html_safe %></p>
-
-      <h3><%= t '.aspects' %></h3>
-      <p><%= t '.aspects_ex1' %></p>
-      <p><%= t('.aspects_ex2').html_safe %></p>
+  <div class="row top_spacing">
+    <div class="span2">&nbsp;
+    </div>
+    <div class="span12">       
+        <p><%= t('.interface1').html_safe %></p>
+		<p class="more-info"><%= t '.interface2' %></p>
+        <p><%= t '.interface3' %></p>
+        <p><%= t '.header' %></p>
+        <div align="center"><%= image_tag "dropdown.png" %></div>
```

Comment:
Thanks. Just spotted this myself, and now fixed. Testing process not up to scratch (it's been so long since I ran any of this on my server...)